flink/flink-scala-parent/src/main/scala/org/apache/zeppelin/flink/FlinkScalaInterpreter.scala

```diff
@@ -843,6 +843,18 @@ abstract class FlinkScalaInterpreter(val properties: Properties,
     })
   }
 
+  private def isYarnUseProxy(): Boolean = {
+    properties.getProperty("flink.webui.yarn.useProxy", "false").toBoolean
+  }
+
+  private def getJmWebUrlUnderProxy(): String = {
+    // for some cloud vender, the yarn address may be mapped to some other address.
+    val yarnAddress = properties.getProperty("flink.webui.yarn.address")
+    if (StringUtils.isNotBlank(yarnAddress)) {
```

Comment:
no return value if it is false

Reply:
Sorry to fix it late.
Now, return option.empty if it is false.